samples/snippets/iam_test.py

```diff
@@ -110,9 +110,17 @@ def test_check_topic_permissions(topic, capsys):
 
 
 def test_check_subscription_permissions(subscription, capsys):
-    iam.check_subscription_permissions(PROJECT, SUBSCRIPTION)
+    iam.check_subscription_permissions(PROJECT_ID, SUBSCRIPTION_ID)
 
     out, _ = capsys.readouterr()
 
     assert subscription in out
     assert "pubsub.subscriptions.consume" in out
+
+
+def test_detach_subscription(subscription, capsys):
+    iam.check_subscription_permissions(PROJECT_ID, SUBSCRIPTION_ID)
```

Comment:
The pubsub.topics.detachSubscription permission is on the topic, not the subscription actually.

Reply:
Ooh, I forgot to update here. Fixed.